### PR TITLE
[dev] fix infer bug of s2anet

### PR DESCRIPTION
**Pull request by liuhuiCNN**
(1) fix background in s2anet lables when deploy/python/infer
(2) fix doc

### CODE REVIEW

File: deploy/python/infer.py [hunk 1]
```diff
@@ -283,6 +283,10 @@ def __init__(self, model_dir):
         self.preprocess_infos = yml_conf['Preprocess']
         self.min_subgraph_size = yml_conf['min_subgraph_size']
         self.labels = yml_conf['label_list']
+        if self.arch == 'S2ANet':
```
Comment (yghstill) [inline deploy/python/infer.py]: mv background condition to export_utils or category.py

Comment (liuhuiCNN) [reply deploy/python/infer.py]: move to ppdet/engine/export_utils.py

File: deploy/python/infer.py [hunk 2]
```diff
@@ -283,6 +283,10 @@ def __init__(self, model_dir):
...
+            # TODO: move background to num_classes
+            if self.labels[0] != 'background':
+                self.labels.insert(0, 'background')
```
Comment (qingqing01) [inline deploy/python/infer.py]: 这样只能解决Python推理问题，C++推理呢？如果需要background类，为什么不在直接导出的标签里提供呢~

Comment (liuhuiCNN) [reply deploy/python/infer.py]: ppdet/engine/export_utils.py done

**Review by heavengate — APPROVED** [review]

**Pull request merged**